### ISSUE TITLE
App is crashing on stop.

### ISSUE THREAD

**Issue by robertlevonyan** [HBiSoft/HBRecorder#51]
**Description**
The app is crashing from your library. The crashlog is from Firebase Crashlytics

**Log**
```
Fatal Exception: java.lang.RuntimeException: Unable to stop service com.hbisoft.hbrecorder.ScreenRecordService@bf32d6: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.media.projection.MediaProjection.stop()' on a null object reference
       at android.app.ActivityThread.handleStopService(ActivityThread.java:4137)
       at android.app.ActivityThread.access$1900(ActivityThread.java:220)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1897)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7397)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)
```

It cannot be reproduced, because I don't have those devices.

**HBRecorder version:** 0.1.14

**Device information**
Nokia N9 - Android SDK 28

**No screenshots**

**Comment by HBiSoft** [comment]
I believe this was fixed in `0.1.15`. This was reported in #44 and fixed in https://github.com/HBiSoft/HBRecorder/releases/tag/0.1.15

Please update your version and let me know if the issue persists.

**Comment by HBiSoft** [comment]
Closing until you confirm the above.
